lib/excon/constants.rb

```diff
@@ -119,10 +119,10 @@ module WaitWritable; end
     :idempotent           => false,
     :instrumentor_name    => 'excon',
     :middlewares          => [
+      Excon::Middleware::Instrumentor,
```

Comment:
Additionally, this perhaps lends additional credence to: https://github.com/excon/excon/issues/529